index.qmd

```diff
@@ -1,55 +1,224 @@
+        d="M0 487 C108.32 487 672.77 486.499 783.26 486.499C893.75 486.499 943.04 486.499 1051.36 486.499C1135.3 486.499 1218.74 489.327 1241.95 469.982C1293.57 426.95 1311.015 327.407 1328.845 262.111C1356 162.661 1377.725 8 1409.818 8C1441.911 8 1472.715 193.386 1490.888 262.111C1509.06 330.837 1526.835 423.687 1568.803 460.973C1600.671 489.286 1631.508 486.499 1722.851 486.499C1816.509 486.499 1890.44 486.499 1991.446 486.499C2092.453 486.499 2198.993 486.499 2300 486.499"/>
+    </svg>
+  </div>
+</div>
+<span class="display-6 d-block text-center py-5 display-md-5 display-lg-4">
+    Bayesian inference with probabilistic programming
+</span>
+<div class="d-flex flex-column align-items-center gap-0 pb-5">
+  <div class="d-flex flex-row flex-wrap panel-wrapper gap-2">
+    <a href="https://turinglang.org/docs/tutorials/docs-00-getting-started/" class="button--fill btn">
+      Tutorials
+    </a>
+    <a href="https://julialang.slack.com/archives/CCYDC34A0" class="button btn">
+      Slack
+    </a>
+    <a href="https://discourse.julialang.org/c/domain/probprog/48" class="button btn">
+      Discourse
+    </a>
+    <a href="https://github.com/TuringLang" class="button btn">
+      GitHub
+    </a>
+  </div>
+</div>
+```
 
-# Turing.jl: Bayesian inference with probabilistic programming. 
+```{=html}
+<div class="d-flex flex-row flex-wrap panel-wrapper gap-4 section-end-space">
+  <div class="panel">
+    <div class="panel-title pb-1">
+      Intuitive
+    </div>
+    Turing models are easy to write and communicate — syntax is close to mathematical notations. 
+  </div>
+  <div class="panel">
+    <div class="panel-title pb-1">
+      General-purpose
+    </div>
+    Turing supports models with discrete parameters and stochastic control flow. 
+  </div>
+  <div class="panel">
+    <div class="panel-title pb-1">
+      Modular & composable
+    </div>
+    Turing is modular, written entirely in Julia, and is interoperable with the powerful Julia ecosystem.
+  </div>
+</div>
+```
 
-### Intuitive
-Turing models are easy to write and communicate — syntax is close to mathematical notations. 
+<div class="d-flex flex-row flex-wrap panel-wrapper gap-3 pb-2">
+<div class="example-text" style="text-align:right;padding:0.5rem;">
+  <div class="fs-4 fw-bold pb-1">
+  Hello, World in Turing
+  </div>
+  Some text about how easy it is to [get going](https://turinglang.org/docs/tutorials/00-introduction/).
+</div>
+<div class="example-code">
+```julia
+@model function coinflip(; N::Int)
+    # Prior belief about the probability of heads
+    p ~ Beta(1, 1)
 
-### General-purpose
-Turing supports models with discrete parameters and stochastic control flow. 
+    # Heads or tails of a coin are drawn from `N`
+    # Bernoulli distributions with success rate `p`
+    y ~ filldist(Bernoulli(p), N)
 
-### Modular and composable 
-Turing is modular, written entirely in Julia, and is interoperable with the powerful Julia ecosystem. 
+    return y
```

Comment:
```suggestion
```

When I got into Turing I personally found return values to be rather confusing. Basic sampling doesn't make use of the return value (you could `return nothing`, or just omit the return statement, and `sample(coinflip; 100) | (y=y,)` still works exactly the same way).

I would prefer to remove it because including it implies that it is needed for the example to work